src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -343,19 +343,33 @@ def k8s_install_connector(client, name, resource_group, connector_name,
     # Get the TenantID
     profile = Profile()
     _, _, tenant_id = profile.get_login_credentials()
+    # Check if we want the linux connector
+    if os_type.lower() in ['linux', 'both']:
+        _helm_install_aci_connector(url_chart, connector_name, service_principal, client_secret,
+                                    subscription_id, tenant_id, rgkaci.name, location,
+                                    node_prefix + '-linux', "Linux")
```

Comment:
Typically we prefer single quote strings unless double quote is needed, like if the string actually contains a single quote.